Gemfile.lock

```diff
@@ -152,12 +152,13 @@ GEM
     lumberjack (1.0.9)
     mail (2.6.3)
       mime-types (>= 1.16, < 3)
-    mas-rad_core (0.0.17)
+    mas-rad_core (0.0.19)
       active_model_serializers
       geocoder
       http
       pg
       rails (~> 4.2.0)
+      redis
```

Comment:
Seems as though that's down to `sidekiq`, since it's just one patch level I think I'll leave it for now.